src/Propel/Generator/Model/Table.php

```diff
@@ -784,6 +784,25 @@ public function setupReferrers($throwErrors = false)
                 }
             }
 
+            // check for incomplete foreign key references when foreign table
+            // has a composite primary key
+            if ($foreignTable->hasCompositePrimaryKey()) {
```

Comment:
`$foreignTable` can be `null` here. I reproduced it with a table defined in schema that had a foreign key to a table not defined in schema but still created with a behavior

Reply:
I believe that is an edge-case. If you have a solution please contribute with a merge-request.

Reply:
@prgTW I have a merge request that infact solves this.

Reply:
actually my first solution I was testing was to compare against `null` as You already did in Your PR. I didn't come up with a better solution so far